src/main/java/org/fundacionjala/sfdc/pageobjects/Contacts/SFCMainPage.java

```diff
@@ -0,0 +1,77 @@
+package org.fundacionjala.sfdc.pageobjects.Contacts;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.pageobjects.SalesForceConnection;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class SFCMainPage extends SalesForceConnection {
+    private WebDriverWait wait;
+
+    @FindBy(how = How.CSS, using = "input[name='new']")
+    private WebElement newContactButton;
+
+    @FindBy(how = How.CSS, using = "tr.dataRow.even.first.dataRow >th[scope='row'] > a")
+    private WebElement lastContactLink;
+
+    @FindBy(how = How.CSS, using = "h2.pageDescription")
+    private WebElement homeContact;
+
+
+    /**
+     * Constructor.
+     */
+    public SFCMainPage() {
+        super();
+        PageFactory.initElements(this.webDriver, this);
+    }
+
+    /**
+     * Method to do click "new" button.
+     * @return a new contact button element.
+     */
+    public WebElement getNewButton() {
+        wait = DriverManager.getInstance().getWaiter();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)